playwright/tests/response-modification.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { expect, test } from '@playwright/test';
-import { StatusCodes } from 'http-status-codes';
+import * as statusCodes from '../data/status-codes.json';
```

Comment:
Why do you need custom data here?

Reply:
It is not needed, I removed it. Thank you!